### PR TITLE
Update codegen to include only NIO implementation.

### PR DESCRIPTION
**Pull request by glbrntt**
- Also regenerated clients/servers to reflect this change (i.e. the removal of `_NIO` from type names).
- Also fixed some Swift 5 warnings re: deprecated APIs in Sources/protoc-gen-swiftgrpc/io.swift

### CODE REVIEW

**Comment by rebello95** [comment]
I'll let @MrMage review in detail - but can we also make sure that the options supported in the README are updated accordingly?

**Comment by glbrntt** [comment]
Sure - the whole README needs a rewrite but I’ll update the codegen part of it as part of this PR.

File: Examples/Google/NaturalLanguageNIO/RUNME [hunk 1]
```diff
@@ -23,7 +23,6 @@ protoc \
 	-I../common/include \
 	--swift_out=googleapis \
 	--swiftgrpc_out=googleapis \
```
Comment (MrMage) [inline Examples/Google/NaturalLanguageNIO/RUNME]: Need to remove the `\` as well?

File: Sources/protoc-gen-swiftgrpc/Generator.swift [hunk 2]
```diff
@@ -106,35 +99,14 @@ class Generator {
...
-        for service in file.services {
-          self.service = service
-          printClient(asynchronousCode: options.generateAsynchronous,
-                      synchronousCode: options.generateSynchronous)
-        }
-      }
-
-      if options.generateTestStubs {
-        if options.generateNIOImplementation {
-          fatalError("Generating test stubs is not yet supported for SwiftGRPC-NIO.")
```
Comment (MrMage) [inline Sources/protoc-gen-swiftgrpc/Generator.swift]: I think this is still true. (At least we will probably need to generate client test stubs at some point.)

File: Sources/protoc-gen-swiftgrpc/Generator.swift [hunk 3]
```diff
@@ -106,35 +99,14 @@ class Generator {
...
-        }
+      for service in file.services {
+        self.service = service
+        printClient()
       }
     }
     println()
     
     if options.generateServer {
-      if options.generateTestStubs && options.generateNIOImplementation {
```
Comment (MrMage) [inline Sources/protoc-gen-swiftgrpc/Generator.swift]: Any idea why this was in here? I did think we have server test stubs already. Or do we have them, but don't need to generate any code for them?

File: Sources/protoc-gen-swiftgrpc/Generator.swift (hunk 2 above)
Comment (glbrntt) [reply Sources/protoc-gen-swiftgrpc/Generator.swift]: It is true, but I didn't find any NIO test stubs so I thought it was better to remove the option than have an option which didn't do anything!

File: Sources/protoc-gen-swiftgrpc/Generator.swift (hunk 3 above)
Comment (glbrntt) [reply Sources/protoc-gen-swiftgrpc/Generator.swift]: No idea, git tells me you wrote this line though, so you may be better positioned to answer that question! 🙃

As far as I can tell there are no test stubs for NIO.

File: Examples/Google/NaturalLanguageNIO/RUNME (hunk 1 above)
Comment (glbrntt) [reply Examples/Google/NaturalLanguageNIO/RUNME]: good catch, fixed!

File: Sources/protoc-gen-swiftgrpc/Generator.swift (hunk 3 above)
Comment (MrMage) [reply Sources/protoc-gen-swiftgrpc/Generator.swift]: Right, I think the test stub so far simply has you use `UnaryResponseCallContextTestStub` and the like directly; no need to generate code for these.

**Pull request merged**